oauthenticator/google.py

```diff
@@ -314,7 +314,7 @@ async def check_allowed(self, username, auth_model):
         # users should be explicitly allowed via config, otherwise they aren't
         return False
 
-    def _service_client_credentials(self, scopes, user_email_domain):
+    async def _service_client_credentials(self, scopes, user_email_domain):
```

Comment:
This function doesnt seem to be relevant to have async, because it doesnt await anything.

Reply:
Fixed!